examples/thresholds/README.md

```diff
@@ -0,0 +1,111 @@
+                 echo "Skipping comment, absolute percentage change is less than 1"
+                 exit 0
+               fi
+   
+               infracost comment github --repo $BASE_REPO_OWNER/$BASE_REPO_NAME --pull-request $PULL_NUM --path /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json --github-token $GITHUB_TOKEN
+   ```
+4. Restart the atlantis application with the new env vars and config
+5. Send a pull request in GitHub to change something in TF, the Infracost pull request comment will be added when you go above your set threshold.
+
+## Running with Gitlab
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) dockerhub image
+2. You'll need to pass the following custom env vars into the container
+   ```sh
+   GITLAB_TOKEN=<your-gitlab-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+   workflows:
+     terraform-infracost:
+       plan:
+         steps:
+           - init
+           - plan
+           # Run Infracost breakdown and save to a tempfile, namespaced by this project, PR, workspace and dir
+           - run: infracost breakdown --path=$PLANFILE --format=json --log-level=info --out-file=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
+           - run: |
+               # Read the breakdown JSON and get costs
+               past=$(cat /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json | jq -r "(.pastTotalMonthlyCost // 0) | tonumber")
+               current=$(cat /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json | jq -r "(.totalMonthlyCost // 0) | tonumber")
+               cost_change=$(cat /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json | jq -r "(.diffTotalMonthlyCost // 0) | tonumber")
+
+               percent_change=999 # default to a high number so we post a comment if there's no past cost
+
+               if [ "$past" != "0" ]; then
+                 percent_change=$(echo "100 * (($current - $past) / $past)" | bc -l)
+               fi
+
+               absolute_percent_change=$(echo "${percent_change#-}")
+               
+               echo "past: ${past}"
+               echo "current: ${current}"
+               echo "cost_change: ${cost_change}"
+               echo "absolute_cost_change: ${cost_change#-}"
+               echo "percent_change: ${percent_change}"
+               echo "absolute_percent_change: ${absolute_percent_change}"
+   
+               if [ $(echo "$percent_change < 1" | bc -l) == "1" ]; then           # Only comment if cost changed by more than plus or minus 1%
```

Comment:
this is funky but is the only way I could get it to evaluate correctly as these numbers can be floats.

Reply:
Should we use a different number other than `1` for the percentage change to make it clearer which is which? Or move that into an variable called `threshold` or something?

Reply:
👍